notebooks/2_PrincipalCovariatesRegression.ipynb

```diff
@@ -968,7 +968,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.8"
+   "version": "3.6.9"
```

Comment:
We should omit these sorts of commits where possible.

Reply:
Sure, but unless we all ensure that we all use the exact same Python version, I believe this is unavoidable, as the version "updates" are done automatically by the Jupyter notebook. In my view, there's no point in worrying about minutiae like this.